README.md

```diff
@@ -41,112 +42,98 @@ conctr <- Conctr(APP_ID, API_KEY, MODEL);
+**Note** Changing the device ID after data has already been set previously will create a new device in Conctr. There will be no link between any data from this newly created device and the device data linked to the previous device ID.
 
 | Key | Data Type | Required | Default Value | Description |
-| ----| --------------- | --------- | ----------- |----------- |
-| *deviceId* | String | No | `imp.configparams.deviceid` | Custom unique identifier that Conctr should store data against for this device. |
+| --- | --------- | -------- | ------------- | ----------- |
+| *deviceId* | String | No | `"imp.configparams.deviceid"` | Custom unique identifier that Conctr should store data against for this device |
 
-##### Example
+#### Example
 
 ```squirrel
-const CUSTOM_DEVICE_ID="device-1";
+const CUSTOM_DEVICE_ID = "device-1";
 
 conctr.setDeviceId(CUSTOM_DEVICE_ID);
 ```
 
 ### sendData(*payload[, callback]*)
 
-The *sendData()* method sends a data payload to Conctr via the data ingeston endpoint. It is called by the data event listener when the device sends data using the device Conctr class. It can also be used directly to send data to Conctr via the agent alone.
+The *sendData()* method sends a data payload to Conctr via the data ingeston endpoint. It is called by the data event listener when the device sends data using the Conctr device class. It can also be used directly to send data to Conctr via the agent alone.
 
-| Key | Data type | Required | Description |
-| ----| --------------- | --------- | ----------- |
-| *payload* | Table | Yes | A table containing the data to be sent to Conctr. This keys in the table should correspond to fields from the model and the keys should be of type specified in the model.|
-| *callback* | Function | No | Function to be called on response from Conctr. function should take two arguements, error and response. When no error occurred the first arguement will be null.|
+| Key | Data Type | Required | Description |
+| --- | --------- | -------- | ----------- |
+| *payload* | Table | Yes | A table containing the data to be sent to Conctr. The keys in the table should correspond to fields from the model and the keys should be of type specified in the model |
+| *callback* | Function | No | Function to be called on response from Conctr. The function should take two arguements, *error* and *response*. When no error occurred, the first arguement will be null |
 
-##### Example
+#### Example
 
 ```squirrel
-local curTempAndPressure = { "temperature" : 29, "pressure" : 1032};
+local currentTempAndPressure = { "temperature" : 29, "pressure" : 1032};
 
-conctr.sendData(curTempAndPressure, function(error, response) {
+conctr.sendData(currentTempAndPressure, function(error, response) {
     if (error) {
-        //handle error
+        // Handle error
```

Comment:
Remove this comment and replace with an error log.
server.error("Failed to deliver to Conctr: " + error);